AirCasting/SessionViews/ExternalSessionsViews/ExternalSessionCard.swift

```diff
@@ -57,6 +57,10 @@ struct ExternalSessionCard: View {
                 mapStatsDataSource?.dataSource.stream = newStream
                 uiStateHandler.changeSelectedStream(sessionUUID: session.uuid, newStream: newStream?.sensorName ?? "")
             })
+            .onChange(of: isMapButtonActive) { _ in
+                reorderButton.navigationLink(isActive: isMapButtonActive)
+                searchAndFollowButton.navigationLink(isActive: isMapButtonActive)
```

Comment:
Maybe we could change the naming here to sth like: `reorderButton.setVisability(to: isVisible)` by taking `isVisible in`? I am not sure about `isVisible` but nothing else came to my mind.